src/kubernetes/server/server-config.yaml

```diff
@@ -37,8 +37,19 @@ data:
               kubernetes:
                 accounts:
                   default:
+                    imagePullSecret: harborsec
```

Comment:
understood "do not merge", noting for updating -- probably want to remove this. we may consider "imagePullSecrets" if theres a usecase for multiple